README.md

```diff
@@ -223,7 +226,10 @@ array = [
 Sidekiq::Cron::Job.load_from_array array
 ```
 
-Bang-suffixed methods will remove jobs that are not present in the given hash/array, update jobs that have the same names, and create new ones when the names are previously unknown.
+Bang-suffixed methods will
+remove jobs where source is `schedule` and are not present in the given hash/array,
+update jobs that have the same names,
+and create new ones when the names are previously unknown.
```

Comment:
Why this break lines here?

Reply:
I am used to the [ventilated prose](https://vanemden.wordpress.com/2009/01/01/ventilated-prose/) writing style, I can revert this if need be.

Reply:
Yes, nothing against this style, but it will be more consistent with the rest of the docs.

Reply:
Have removed this styling.